app/cli/cmd/attestation_init.go

```diff
@@ -38,6 +39,13 @@ func newAttestationInitCmd() *cobra.Command {
 		Annotations: map[string]string{
 			useWorkflowRobotAccount: "true",
 		},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if os.Getenv(robotAccountEnvVarName) != "" && workflowName != "" {
```

Comment:
One of my concerns is that the API token will be set on the env variable $CHAINLOOP_ROBOT_ACCOUNT because that's how it's named on the `attestation` command so this print will be always shown on the CLI when the `workflow_name` is set.

Is that what we want, to always show the message on API Tokens?